cmd/ci/main.go

```diff
@@ -10,11 +10,23 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
-type TestGroup struct {
-	Name   string   `json:"name"`
-	Runner []string `json:"runner"`
+type TestGroupExecutionContext struct {
+	Name   string `json:"name"`
+	Runner Runner `json:"runner"`
 }
 
+type Runner []string
```

Comment:
Maybe add the explanatory comment for why Runner is `[]string` rather than just `string`?